frontend/app/views/fragments/form/subscriberNumber.scala.html

```diff
@@ -3,12 +3,23 @@
 <fieldset class="fieldset">
 
     <div class="fieldset__heading">
-        <h2 class="fieldset__headline">Subscribers pay nothing for 6 months</h2>
+        <h2 class="fieldset__headline">Subscribers pay nothing<br /> for six months</h2>
         <div class="fieldset__note">
-            <p>Need help redeeming this offer?</p>
-            <p>Email @Email.membershipSupport</p>
-            <p>Call 0330 333 6898</p>
-            <p>(Lines open 08:00 to 17:30 Mon - Fri, 08.30 to 12.30 weekends)</p>
+            <button class="u-button-reset js-toggle"
+                    data-toggle-label="Hide"
+                    data-toggle="js-sub-offer-help"
+                    data-metric-trigger="click"
+                    data-metric-category="registration"
+                    data-metric-action="toggle-subscriber-help"
+                    data-toggle-icon="triangle-up">
+                @fragments.inlineIcon("triangle-down", Seq("u-align-right"))
+                Need help redeeming this offer?
+            </button>
+            <div id="js-sub-offer-help" data-toggle-hidden>
```

Comment:
I'd probably remove the 'js-' prefix here as it's an ID and presumably it's only ever referenced in the `data-toggle` above.